src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -613,6 +613,75 @@ public void testLastIndexOfLastBoundaries() {
         Assert.assertEquals(result, expected);
     }
 
+    @Test
+    public void testLastIndexOfOneMismatchLastBoundaries() {
+        final String reference = "AAAACCCCTTTTGGGG";
+
+        // match right boundary of reference
+        String query = "TGAGG";
+        int result = Utils.lastIndexOfOneMismatch(reference.getBytes(), query.getBytes());
+        int expected = 11;
```

Comment:
Write this as `reference.length - expected.length` so it's not a magic constant.